src/app/components/courses-page/courses-page.component.ts

```diff
@@ -1,10 +1,31 @@
-import { Component } from '@angular/core';
+import { Component, OnChanges, OnInit, SimpleChanges } from '@angular/core';
+
+import { courses } from 'src/app/shared/mock/courses';
+import { ICourse } from 'src/app/shared/interfaces/course';
```

Comment:
Generally it is not a very good practice to import from src folder, but you may use aliases to optimize the import path
Please read about them and use :)
[Alias](https://medium.com/@thepawanluhana/angular-alias-import-with-typescript-d8e79dd9e5d)

Reply:
Thanks for the info about aliases!